tests/HasPermissionsTest.php

```diff
@@ -234,15 +234,15 @@ public function it_throws_an_exception_when_the_permission_does_not_exist()
     {
         $this->expectException(PermissionDoesNotExist::class);
 
-        $this->testUser->hasPermissionTo('does-not-exist');
+        $this->testUser->hasPermissionTo(7892);
```

Comment:
Were the two changes to a numeric permission name intentional?

Reply:
My mistake. I forgot to undo that. Will change.